plugin/core/views.py

```diff
@@ -385,7 +396,8 @@ def minihtml(view: sublime.View, content: Union[str, Dict[str, str], list], allo
     if is_plain_text:
         return text2html(result)
     else:
-        frontmatter_config = mdpopups.format_frontmatter({'allow_code_wrap': True})
+        frontmatter_config = mdpopups.format_frontmatter(
+            {'allow_code_wrap': True})
```

Comment:
The line wasn't that long. Why extra line break?

Reply:
I cannot trust pyls to pick up the formatting settings in tox.ini apparently. I reverted these formatting changes.